voluncheer/profiles/views/activate_email.py

```diff
@@ -18,9 +18,7 @@ def activateEmail(request, user, to_email):
         {
             "email": user.email,
             "user": user,
-            "domain": "voluncheer-main.us-east-1.elasticbeanstalk.com"  # noqa E501
-            if os.getenv("IS_PRODUCTION")
-            else "127.0.0.1:8000",  # noqa E501
+            "domain": os.getenv("DOMAIN"),  # noqa E501
```

Comment:
Add a prefix to the environment variable related to the use of the variable, for instance `SES_SENDER_DOMAIN` or something like that. That way when we are _only_ looking at the environment variable we can tell what it is used for.

We should also pull these environment variables when the file is initialized (or ideally in a shared file during initialization) to avoid having to read them every time these functions are used. By reading them during initialization they are only read once and then re-used as long as the server remains alive. All in all it is much more efficient.

Here and elsewhere.

Reply:
One thing still not done is moving these environment reads from within the function to module initialization.

Reply:
If you don't want to do it now at least add a TODO comment so we can keep track of what we still want to improve.

Reply:
Never mind I think I have an idea!

Reply:
Yes, perfect.